nnpdf_data/nnpdf_data/commondata/ATLAS_Z0J_8TEV/kinematics_legacy_PT-Y.yaml

```diff
@@ -0,0 +1,1441 @@
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 9.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 25.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 49.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 81.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 132.25
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 210.25
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 324.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 506.25
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 756.25
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 1122.25
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 1681.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 2500.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 3600.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 4900.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 6400.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 9025.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 16256.25
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 30625.0
+    max: null
+  k3:
+    min: null
+    mid: 8000.0
+    max: null
+- k1:
+    min: null
+    mid: 2.2
+    max: null
+  k2:
+    min: null
+    mid: 302500.0
```

Comment:
@enocera @scarlehoff I don't understand why, in the old implementation, the mid value of the bin is squared, even though the x-sec is differential in $p_T$ and not in $(p_T)^2$. For instance, the last bin in $p_T$ in hepdata is $[200,900]$ GeV, while here we have $(550)^2$. Is there a subtle reason for that? And most importantly, should I re-implement the kinematics using $p_T$ or $(p_T)^2$?

<img width="657" alt="Screenshot 2024-10-10 at 18 55 06" src="https://github.com/user-attachments/assets/fa87b689-a205-42aa-947a-192eee9e4a11">

Reply:
Because these k2 would get a sqrt() somewhere in plotoptions.

Just select a process option from here https://github.com/NNPDF/nnpdf/blob/master/validphys2/src/validphys/process_options.py (or implement a new one). My vote is to make sure the process type in process options accepts pT and use pT since that's what we plot